app/Policies/PronounsPolicy.php

```diff
@@ -68,33 +90,61 @@ public function create(Authenticatable $authenticatable): Response
 
     public function update(Authenticatable $authenticatable, Pronouns $model): Response
     {
+        if (SettingsPermissions::active()) {
+            return $authenticatable->canOrElse(
+                abilities: 'settings.*.update',
+                denyResponse: 'You do not have permission to update these pronoun.'
+            );
+        }
+
         return $authenticatable->canOrElse(
             abilities: ["product_admin.{$model->getKey()}.update"],
-            denyResponse: 'You do not have permission to update these pronouns.'
+            denyResponse: 'You do not have permission to update these pronoun.'
         );
     }
 
     public function delete(Authenticatable $authenticatable, Pronouns $model): Response
     {
+        if (SettingsPermissions::active()) {
+            return $authenticatable->canOrElse(
+                abilities: 'settings.*.delete',
+                denyResponse: 'You do not have permission to delete these pronoun.'
+            );
+        }
+
         return $authenticatable->canOrElse(
             abilities: ["product_admin.{$model->getKey()}.delete"],
-            denyResponse: 'You do not have permission to delete these pronouns.'
+            denyResponse: 'You do not have permission to delete these pronoun.'
         );
     }
 
     public function restore(Authenticatable $authenticatable, Pronouns $model): Response
     {
+        if (SettingsPermissions::active()) {
+            return $authenticatable->canOrElse(
+                abilities: 'settings.*.restore',
+                denyResponse: 'You do not have permission to restore these pronoun.'
+            );
+        }
+
         return $authenticatable->canOrElse(
             abilities: ["product_admin.{$model->getKey()}.restore"],
-            denyResponse: 'You do not have permission to restore these pronouns.'
+            denyResponse: 'You do not have permission to restore these pronoun.'
         );
     }
 
     public function forceDelete(Authenticatable $authenticatable, Pronouns $model): Response
     {
+        if (SettingsPermissions::active()) {
+            return $authenticatable->canOrElse(
+                abilities: 'settings.*.force-delete',
+                denyResponse: 'You do not have permission to permanently delete these pronoun.'
+            );
+        }
+
         return $authenticatable->canOrElse(
             abilities: ["product_admin.{$model->getkey()}.force-delete"],
-            denyResponse: 'You do not have permission to permanently delete these pronouns.'
+            denyResponse: 'You do not have permission to permanently delete these pronoun.'
```

Comment:
Please review the PR policies yourself and revert any changes you have made to the messages - including the `settings.` and `product_admin.` variants. The old grammar used was correct